packages/objects-side-by-side/_side-by-side.scss

```diff
@@ -0,0 +1,52 @@
+// ==========================================================================
+// SideBySide Object
+// ==========================================================================
+
+//
+// Scales namespace variable
+///
+$scales-namespace: null !default;
+
+//
+// Class level namespacing variable
+///
+$scales-objects-class-namespace: null !default;
+
+//
+// Available Variables
+///
+$side-by-side-gutter: 1rem !default;
+
+//
+// Default SideBySide
```

Comment:
I see this written as "Side by Side" in some place and "SideBySide" in others. A 3rd option would be to write it as "Side-by-side". I think I would just use "Side by Side" everywhere.